static/src/javascripts/projects/commercial/modules/dfp/load-advert.js

```diff
@@ -11,7 +12,10 @@ export const loadAdvert = (advert: Advert): void => {
             advert.startLoading();
             return prebid.requestBids(advert);
         })
-        .then(() => window.googletag.display(advert.id));
+        .then(() => {
+            markTime(`Commercial: Ad loaded: ${advert.id}`);
+            return window.googletag.display(advert.id);
```

Comment:
Not sure this works. There's a callback you can register with googletag that is fired when an Ad is effectively loaded. It's already used for something else.